build/configs/qemu/tash_16m/Make.defs

```diff
@@ -137,5 +137,5 @@ HOSTCFLAGS = -Wall -Wstrict-prototypes -Wshadow -g -pipe
 HOSTLDFLAGS =
 
 define DOWNLOAD
-	@gnome-terminal -e "qemu-system-arm -M lm3s6965evb -kernel $(TOPDIR)/../build/output/bin/tinyara -nographic"
+	@gnome-terminal -e "qemu-system-arm -M lm3s6965evb -kernel $(TOPDIR)/../build/output/bin/tinyara -nographic -gdb tcp::3333"
```

Comment:
yes, i thought of "make run" would be good for executing tinyara on qemu hardware. 
But as you mentioned, then we need to support "make run" for other boards as well and many commands can cause confusion.
What do you suggest?